Makefile

```diff
@@ -8,11 +8,21 @@ MANCHAPTER=Foreman
 TMPDIR=local-tmp-foreman
 
 ifndef DISTRO
-$(error Set the DISTRO variable e.g. rhel7 or fedora21)
+$(error *** Set the DISTRO variable e.g. rhel7 or fedora21 ***)
+endif
+
+ifneq ("$(wildcard /usr/share/selinux/devel/include/services/docker.if)","")
```

Comment:
Good catch, also changed the warning to hard failure when docker is not present.